### ISSUE TITLE
Enterprise N still doesn't have the WSL feature

### ISSUE THREAD

**Issue by xornand** [Microsoft/BashOnWindows#334]
On the developer track, with Insider Hub, logged in with my MSDN account, the WSL feature isn't provided as an option. Is there a non-linear way to brute-install the feature (perhaps via PS?). Pardon the ignorance, I'm primarily a *nix engineer so this is foreign territory for me.

**Comment by benhillis** [comment]
What build number are you on?  Windows 10 Anniversary Edition - Build 14316.  Please see our [installation guide](https://msdn.microsoft.com/en-us/commandline/wsl/install_guide) for more details.

**Comment by hut8** [comment]
After you get on the "Fast track", you have to restart -- this tripped me up a bit, but after a reboot it appeared.

**Comment by russalex** [comment]
I was unaware that we ran the Enterprise N builds in Insiders.  Good to know you're running it.  
Here are a few things to remember when installing:

1. We are x64 only.  x86 builds of Enterprise N will not show the WSL as an optional component
2. There was at least 1 N build which did not allow you to turn on the developer option.  There are instructions [here ](https://msdn.microsoft.com/en-us/windows/uwp/get-started/enable-your-device-for-development)if you hit this in your build.
3. We start in the same build number for N - 14316.  The [install guide](msdn.microsoft.com/en-us/commandline/wsl/install_guide) Ben points out has instructions on how to check your build number. 

Hope that helps.  I was able to run on Enterprise N back in the beginning of April but I did need to enable developer mode manually following the link in the second bullet point above.
